pyvista/plotting/plotting.py

```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
 
     @property
     def camera_position(self):
-        """Return camera position of the active render window."""
+        """Return camera position of the active render window.
+
+        Examples
+        --------
+        Return camera's position and then reposition it.
+
+        >>> mesh = examples.download_bunny_coarse()
+
+        >>> pl = pv.Plotter()
+        >>> pl.add_mesh(mesh, show_edges=True, color="white")
+        >>> pl.add_points(mesh.points, color="red", point_size=10)
```

Comment:
`add_mesh` and `add_points` returns `vtk.vtkActor`. We need "_" variable.
```suggestion
        >>> _ = pl.add_mesh(mesh, show_edges=True, color="white")
        >>> _ = pl.add_points(mesh.points, color="red", point_size=10)
```